sql/2023/01/third-party-render-blocking-script.sql

```diff
@@ -0,0 +1,60 @@
+# HTTP Archive query to measure the impact of third-party render blocking scripts for WordPress sites.
```

Comment:
The query doesn't really measure any _impact_, it's more about _usage_ of third-party render blocking scripts.

Reply:
Thanks for pointing it out. Comment updated.